Makefile

```
@@ -17,7 +17,7 @@ else
 	PRODUCT:=$(firstword $(PRODUCTS))
 endif
 
-PLATFORM:=$(shell echo -n $(PRODUCT) | cut -d'_' -f2- | sed 's/_\(amd64\|arm.*\)//')
+PLATFORM:=$(shell echo -n $(PRODUCT) | sed 's/_\(amd64\|arm.*\)$//')
```

checkmake (0.2.2)

[warning] Target "PLATFORM" should be declared PHONY.

(phonydeclared)